managed/services/telemetry/telemetry.go

```diff
@@ -106,18 +106,35 @@ func (s *Service) Run(ctx context.Context) {
 	defer ticker.Stop()
 
 	doSend := func() {
+		var settings *models.Settings
+		err := s.db.InTransaction(func(tx *reform.TX) error {
+			var e error
+			if settings, e = models.GetSettings(tx); e != nil {
+				return e
+			}
+			return nil
+		})
+		if settings.Telemetry.Disabled {
+			s.l.Info("disabled via settings")
+			return
+		}
+
 		report, err := s.prepareReport(ctx)
```

Comment:
I think that we should print the report into log stream for debug purposes. this was mainly the idea behind using PERCONA_TEST_TELEMETRY_DISABLE_SEND env var - just to check what exactly is gathered from datasources but do not send it to Portal

Reply:
I understand your intention, but in this case, `PERCONA_TEST_TELEMETRY_DISABLE_SEND` will do more than that it is intended to do. We can always had that info printed in debug.